src/uct/ib/base/ib_iface.c

```diff
@@ -808,6 +808,10 @@ UCS_CLASS_INIT_FUNC(uct_ib_iface_t, uct_ib_iface_ops_t *ops, uct_md_h md,
     self->release_desc.cb           = uct_ib_iface_release_desc;
     self->config.enable_res_domain  = config->enable_res_domain;
     self->config.qp_type            = init_attr->qp_type;
+    self->config.is_global          = config->is_global ||
```

Comment:
Check for `uct_ib_iface_is_roce(self)` can also be included there, wdyt?

Reply:
technically yes, but here is user configuration...
@brminich ideas?

Reply:
> technically yes, but here is user configuration...

yes, I agree. but it can't be false anyway if link type is RoCE

Reply:
fixed